students/manu-lope/Ejercicio variables JS/ejercicio.js

```diff
@@ -0,0 +1,79 @@
+/* 1.-Solicitar al usuario dos variables numéricas (usando prompt) y realizar su suma, 
```

Comment:
Uncomment the code. 
It whould be easy if you split the exercises in different files. one file per exercise inside this folder